### DISCUSSION TITLE
Render Obsidain Markdwonfiles in Quarto

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [quarto-dev/quarto-cli#5449] [Q&A] [answered]
I use obsidian as my note taking app, but mostly when i do ml tasks i want to explore it quarto. Can i apply my markdown files and the linking in an easy way in quarto . 
I use some highlights, emoji ..etc from obsidian and want to render them in quarto !

**Comment by mcanouil** [comment]
I believe this is a duplicate of #3715 and closing this discussion for now (unless it's really something different).

**Comment by cderv** [comment, marked as answer]
Obsidian related discussion https://forum.obsidian.md/t/quarto-support-open-source-scientific-and-technical-publishing-system/32625/36 and Obsidian plugin https://github.com/danieltomasz/qmd-as-md-obsidian

Just for reference for anyone looking for Obsidian and Quarto
